sdk/storage/storage-queue/test/node/emulator-tests.spec.ts

```diff
@@ -20,7 +20,7 @@ describe("Emulator Tests", () => {
     await queueClient.create();
   });
 
-  it("MessagesClient can be created with a connection string and a queue name", async () => {
```

Comment:
Thanks.
We have a separate issue tracking for this - #5593. Please close this too if everything looks good :)

Reply:
This issue will be fixed as part of this PR then. I went out of the original issue's scope by mistake.